es6-shim.js

```diff
@@ -156,7 +156,9 @@
   var globals = getGlobal();
   var globalIsFinite = globals.isFinite;
   var hasStrictMode = (function () { return this === null; }.call(null));
-  var startsWithIsCompliant = startsWithRejectsRegex() && startsWithHandlesInfinity;
+  var startsWithExistAndIsNotCompliant = 
+    (typeof String.prototype.startsWith !== 'undefined') && 
+    (!startsWithRejectsRegex() || !startsWithHandlesInfinity());
```

Comment:
Please put all this on one line - line length limits are just as silly as arbitrary line breaks to accommodate them :-)